frontend/packages/operator-lifecycle-manager/src/style.scss

```diff
@@ -254,3 +254,55 @@ $co-affinity-row-margin: 15px;
 .olm-descriptor__title {
   display: inline-block;
 }
+
+.pf-c-card__body.co-clusterserviceversion-install__gray_background {
+  background-color: #f0f0f0;
+}
+
+#operator-install-page .pf-c-card {
+  margin-bottom: 5px;
+  max-width: 600px;
+  // margin-top: auto;
+}
+
+#operator-install-page .pf-c-alert {
+  margin-bottom: 5px;
+}
+
+#operator-install-page .pf-c-button.co-clusterserviceversion-link__shift_left {
+  padding-top: 10px;
+  padding-left: 0px;
+}
+
+#operator-install-page .pf-c-button.co-clusterserviceversion-button__left_padding {
+  margin-left: 10px;
+}
+
+.co-operator-install-page__bottom-padding {
+  padding-bottom: 10px;
+}
+.co-clusterserviceversion-install__heading {
+  font-weight: bold;
+  padding-bottom: 10px;
+}
+
+.co-clusterserviceversion-install__button_div {
```

Comment:
I don't believe this is needed. 
Button spacing should be controlled by using `<ActionGroup>` 
https://github.com/sg00dwin/console/commit/2d2b4d71b70a8c3aa4bece2a1d2e5db78fe4a9f7#diff-2083c529cf4855092119fb0e8b179b31R262